Dockerfile

```diff
@@ -1,66 +1,97 @@
+FROM ubuntu:18.04 AS builder
+
+LABEL maintainer="team@appwrite.io"
+
+ARG TESTING=false
```

Comment:
We can skip this all tests will run from outside the container

Reply:
Related comments:
- https://github.com/appwrite/appwrite/pull/109#discussion_r349879353
- https://github.com/appwrite/appwrite/pull/109#discussion_r349879353